src/site/content/en/payments/android-payment-apps-delegation/index.md

```diff
@@ -1,36 +1,267 @@
 ---
 layout: post
-title: Support payment method change from an Android payment app
+title: Providing shipping and contact information from an Android payment app
 subhead: |
-  How to update your Android payment app to support payment method change with Web Payments APIs.
+  How to update your Android payment app to provide shipping address and payer's contact information with Web Payments APIs.
 authors:
   - sahel
 date: 2020-07-17
 updated: 2021-09-14
 description: |
-  Learn how to modify your Android payment app to provide the payment method change when the merchant has requested them via the Payment Request API.
+  Learn how to modify your Android payment app to provide the user's selected shipping address as well as contact information when the merchant has requested them via the Payment Request API.
 tags:
   - payments
 feedback:
   - api
 ---
 
-{% Aside 'warning' %}
+Entering shipping address and contact information through a web form can be a
+cumbersome experience for customers. It can cause errors and lower conversion
+rate.
 
-Shipping and address support in [the Payment Request API is removed from the
-specification](https://github.com/w3c/payment-request/pull/955) and is no longer
-functional in Android payment apps.
+That's why the Payment Request API supports a feature to request shipping
+address and contact information. This provides multiple benefits:
 
-{% endAside %}
+* Users can pick the right address with just a few taps.
+* The address is always returned in [the standardized
+  format](https://w3c.github.io/payment-request/#paymentaddress-interface).
+* Submitting an incorrect address is less likely.
+
+This functionality can be deferred to a payment app to offer a unified payment
+experience and it's called *delegation*.
```

Comment:
So just to check, what we're saying is "Payment apps can request shipping address and contact information directly from a customer to provide a unified payment experience in a process called *delegation*."

Is that it? Or is it that "Browsers can defer the collection of shipping address and contact information to a payment app to provide a unified payment experience. This functionality is called *delegation*."

I think it's the latter—if so, I guess using this more specified copy might help contextualise the bit after this. Also is it any browser or Chrome specifically?